packages/babel-plugin-transform-nullish-coalescing-operator/src/index.js

```diff
@@ -0,0 +1,33 @@
+import syntaxNullishCoalescingOperator from "babel-plugin-syntax-nullish-coalescing-operator";
+
+const OPERATOR = "??";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxNullishCoalescingOperator,
+
+    visitor: {
+      LogicalExpression(path) {
+        const { node } = path;
+        if (node.operator !== OPERATOR) {
+          return;
+        }
+
+        const scope = path.scope.parent || path.scope;
```

Comment:
Just `path.scope`.

Reply:
I did this to fix an issue with function default arguments.

```js
function foo(a, b = a ?? 0) {}
```

Would otherwise transform to

```js
function (a, b = (_ref = a, ref != null ? _ref : 0)) {
  var _ref;
}
```

Which leaks a global. Is there a better way to handle this?